orb/src/jobs/deploy-production.yml

```diff
@@ -33,5 +32,5 @@ steps:
       name: Deploy to production
       command: npx dotcom-tool-kit deploy:production
   - change-api/change-log:
-      environment: production
-      system-code: <<parameters.system-code>>
+      environment: << parameters.environment >>
+      system-code: << parameters.system-code >>
```

Comment:
**question:** As we are defining the parameters at the top of the config file, should we be calling the parameter with [`pipeline`](https://circleci.com/docs/pipeline-variables/#pipeline-parameters-in-configuration)?

```suggestion
      environment: << pipeline.parameters.environment >>
      system-code: << pipeline.parameters.system-code >>
```

Reply:
`parameters.system-code` had worked before so I feel confident it is not needed now.